modules/internal/Atomics.chpl

```diff
@@ -187,8 +187,10 @@ module Atomics {
       atomic_init(_v, default);
     }
 
+    pragma "fn synchronization free"
```

Comment:
Why do we need to mark the extern and the real function as sync free? Shouldn't marking the extern be enough?